EpikCord/client/websocket.py

```diff
@@ -248,11 +249,15 @@ async def handle(self, message: DiscordWSMessage):
         if event["op"] != OpCode.DISPATCH:
             await self.opcode_mapping[event["op"]](event["d"])
         else:
+            event_name = event["t"].lower()
+            event_data = event["d"]
+
+            if event_handler := getattr(self, f"_{event_name}", None):
```

Comment:
This sounds like evil. the decorator register everything in a map, let's use it

Reply:
This is for the internal calls